sdcm/cluster.py

```diff
@@ -1839,6 +1840,15 @@ def config_setup(self,
 
             if ldap:
                 scylla_yml.update(self.get_ldap_config())
+            elif ms_ad_ldap:
+                scylla_yml.update({
+                    'ldap_attr_role': 'cn',
+                    'ldap_bind_dn': 'cn=admin,dc=scylla-qa,dc=com',
```

Comment:
Can we make the docker ldap and MS AD to use same domain?
Or add helper function to get the domain?

Here we can make get_ldap_config to output different config according the ldap is used.

Reply:
as the MS AD was already created, i moved the openldap config to be the same as the one being used on MS AD one